### PR TITLE
Makes the header and footer floating around the selection

### PR DESCRIPTION
**Pull request by javierarce**
anim2

Original issue: #4899

### CODE REVIEW

**Comment by javierarce** [comment]
Review this, please please, @xavijam.

File: app/assets/stylesheets/table/export_image_view.css.scss [hunk 1]
```diff
@@ -11,12 +11,18 @@
...
+  top: -35px;
   height: 40px;
-  top: 30px;
   z-index: 105;
 }
+.ExportImageView-Header--inner {
+  position:absolute;
+  width: 400px;
+  left: 50%;
+  margin-left: -200px;
```
Comment (xavijam) [inline app/assets/stylesheets/table/export_image_view.css.scss]: I am not a big friend of negative margins.

File: lib/assets/javascripts/cartodb/table/export_image_view.js [hunk 2]
```diff
@@ -166,6 +163,12 @@ cdb.admin.ExportImageView = cdb.core.View.extend({
 
     this.$(".CanvasExport").css({ height: height, width: width });
 
+    if (width < this._MIN_WIDTH) {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/export_image_view.js]: You can use jQuery ```.toggleClass``` function for the same behaviour.

File: app/assets/stylesheets/table/export_image_view.css.scss (hunk 1 above)
Comment (javierarce) [inline app/assets/stylesheets/table/export_image_view.css.scss]: I need to center the element and make its size independent of the parent width. I think this is the simplest solution.

Comment (xavijam) [inline app/assets/stylesheets/table/export_image_view.css.scss]: Ah, right, i didn't pay attention to the whole thing :+1:

Comment (javierarce) [inline app/assets/stylesheets/table/export_image_view.css.scss]: Thanks!

**Pull request merged**